### DISCUSSION TITLE
Please tell me a singer you like  🎶🎶🎶

### DISCUSSION THREAD

**Discussion by ferris-lucas** [NAIST-SE/AttractingContributors#808] [2023 Attraction Contributions]
Hello!
Please tell me a musical group (or singer) that you like.

Thank you.

Contribute here: https://github.com/ferris-lucas/AttractingContributors-Bands

**Comment by kohei-ts** [comment]
I added PAS TASTA, a J-Pop artist of the new age, whose songs are similar to hyperpop.
Listen to their music if you'd like.

And could you tell me about your favorite rock genre?
https://github.com/kohei-ts/AttractingContributors-MusicGenre

**Reply by ferris-lucas** [reply]
Added. Many thanks!!!

**Comment by Sanaa3131** [comment]
I couldn't merge your pull request because of a branch conflict, do you know how to solve it ? BTW, I have already contributed to your file.

**Reply by ferris-lucas** [reply]
I updated the pull request, can you check it? 

If you have issues with conflicts, you can use the "Resolve Conflict" function ([here is a tutorial](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github))

Let me know if you still have issues to merge it.

**Comment by Sanaa3131** [comment]
Hi Lucas, thank you so much for the tutorial, it was very helpful.
